### PR TITLE
Feature/cloudwatchlogs reorg

### PR DESCRIPTION
**Pull request by ovlords**
Added Log files for Dispatcher, Simian Army and Orchestrator components:
https://github.com/shinesolutions/aem-aws-stack-provisioner/issues/35

### CODE REVIEW

File: ansible/inventory/group_vars/defaults.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ aws:
   aem_artifacts_base: overwrite-me
   aem_certs_base: overwrite-me
   install_ssm_agent: true
+  install_cloudwatchlogs: true
```
Comment (cliffano) [inline ansible/inventory/group_vars/defaults.yaml]: Please update configuration document with this property. https://github.com/shinesolutions/packer-aem/blob/master/docs/configuration.md

File: ansible/templates/hieradata.j2 [hunk 2]
```diff
@@ -3,6 +3,7 @@ timezone::locality: '{{ timezone.locality }}'
 cloudwatchlogs::region: '{{ aws.region }}'
 
 config::base::install_amazon_ssm_agent: {{ aws.install_ssm_agent }}
+config::base::install_cloudwatchlogs: {{ aws.install_cloudwatchlogs }}
```
Comment (cliffano) [inline ansible/templates/hieradata.j2]: Please add inspec test for the existence of cloudwatch configuration file when install cloudwatch flag is set to true.

File: ansible/inventory/group_vars/defaults.yaml (hunk 1 above)
Comment (ovlords) [reply ansible/inventory/group_vars/defaults.yaml]: Done

File: ansible/templates/hieradata.j2 (hunk 2 above)
Comment (ovlords) [reply ansible/templates/hieradata.j2]: Done

**Pull request merged**